bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -4026,9 +4016,6 @@ tourCatalog_view_action_synch_chart_years_tooltip    = Use the same scaling for
 tourCatalog_view_action_synch_charts_byScale_tooltip = Show the marker in the compared tour with the same scale as in the reference tour
 tourCatalog_view_action_synch_charts_bySize_tooltip  = Show the marker in the compared tour with the same size as in the reference tour
 tourCatalog_view_action_undo_marker_position         = Undo changes, set moved marker to original position
-tourCatalog_view_compare_job_subtask                 = Compared tours: {0} / {1}
-tourCatalog_view_compare_job_task                    = Compare Task
-tourCatalog_view_compare_job_title                   = Compare Tours
```

Comment:
@FJBDev How have you found these text keys, "Find Broken Externalizes Strings" is not working since a while, respectively it finds too many wrong text keys

![image](https://user-images.githubusercontent.com/1283445/230788180-4fdf3760-b93d-4716-8d66-5692a8aaa148.png)

![image](https://user-images.githubusercontent.com/1283445/230788299-6857210d-9361-4926-8e04-c7ceee150f57.png)

Reply:
> @FJBDev How have you found these text keys, 

I use the [UCDetector plugin](https://marketplace.eclipse.org/content/unnecessary-code-detector) that is very good at finding unused strings, methods etc....